src/bundle/Resources/public/js/scripts/core/collapse.js

```diff
@@ -1,11 +1,60 @@
-(function (global, doc) {
+(function (global, doc, bootstrap) {
+    let toggleAllTimeout;
+    let singleElementClicked = [];
+    const toggleAllBtn = doc.querySelector('.ibexa-multi-collapse-btn');
+    const expandAll = toggleAllBtn?.querySelector('.ibexa-attribute-group__toggler-expand');
+    const collapseAll = toggleAllBtn?.querySelector('.ibexa-attribute-group__toggler-collapse');
+    const MULTI_COLLAPSE_BODY_CLASS = '.ibexa-multi-collapse';
+
+    const toggleMultiCollapseButton = () => {
+        collapseAll.classList.toggle('d-none');
+        expandAll.classList.toggle('d-none');
+    };
+    const toggleMultiCollapseIfNeeded = () => {
+        const allGroups = doc.querySelectorAll(MULTI_COLLAPSE_BODY_CLASS);
+
+        if (singleElementClicked.length === allGroups.length || singleElementClicked.length === 0) {
+            toggleMultiCollapseButton();
+        }
+    };
+    const handleCollapseAction = (expandAction) => {
+        singleElementClicked = [];
+        doc.querySelectorAll('.ibexa-collapse').forEach((collapseNode) => {
```

Comment:
I think here's some bigger problem - it should be contextualized in some way. Now you'll collapse ALL elements on page that has this class, even if they're in some totally different place and are used for something totally different.

Probably instead of `doc` you should use some container to limit to some specific part of site